modules/product/src/Form/ProductForm.php

```diff
@@ -270,6 +318,32 @@ protected function actions(array $form, FormStateInterface $form_state) {
 
   /**
    * {@inheritdoc}
+   *
+   * Updates the node object by processing the submitted values.
+   *
+   * This function can be called by a "Next" button of a wizard to update the
+   * form state's entity with the current step's values before proceeding to the
+   * next step.
+   */
```

Comment:
It's not allowed to have both `{@inheritdoc}` and documentation in docblock, so we should either remove this comment or copy the relevant portion of the parent's docblock over.